sunpy/time/tests/test_time.py

```diff
@@ -95,6 +95,20 @@ def test_parse_time_individual_numpy_datetime():
     assert isinstance(dt, datetime)
 
 
+def test_parse_time_numpy_datetime_timezone():
+    dt64 = np.datetime64('2014-02-07T16:47:51-0500')
+    dt = parse_time(dt64)
+
+    assert dt == datetime(2014, 2, 7, 21, 47, 51)
+
+
+def test_parse_time_numpy_datetime_ns():
+    dt64 = np.datetime64('2014-02-07T16:47:51.008288000')
+    dt = parse_time(dt64)
+
+    assert dt == datetime(2014, 2, 7, 16, 47, 51, 8288)
+
+
```

Comment:
Actually we are kind of testing that. `dt64 = np.datetime64('2014-02-07T16:47:51.008288000')` has a `dtype` of `ns` not `us`.

Reply:
Done. :smiley:

Reply:
could you add another one to test for rounding?  For example, does 
`dt2_64 = np.datetime64('2014-02-07T16:47:51.008288751')`
get rounded up to `datetime(2014, 2, 7, 16, 47, 51, 8289)` or are the last three digits of the dattime64 ignored so that `datetime(2014, 2, 7, 16, 47, 51, 8288)` are returned?  Thanks!

Reply:
Done. Again. :smiley: